dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/mbeans/PluginGenerator.java

```diff
@@ -208,6 +211,10 @@ protected BundleContext getBundleContext() {
         return FrameworkUtil.getBundle(PluginGenerator.class).getBundleContext();
     }
 
+    public boolean isBundleUninstalled() {
+        return bundle.getState() == Bundle.UNINSTALLED;
+    }
+
```

Comment:
Does this method really need to be public?

Reply:
good catch, will change